sources/packages/api/src-sql/Institution/Create-Institution.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS institutions(
+  id SERIAL PRIMARY KEY, 
+  legal_operating_name VARCHAR(64) NOT NULL, 
+  operating_name VARCHAR(64) NULL,  
+  primary_phone VARCHAR(64) NOT NULL,  
+  primary_email VARCHAR(64) NOT NULL,  
+  institution_type VARCHAR(64) NOT NULL,  
+  website VARCHAR(64) NOT NULL,  
+  regulating_body VARCHAR(64) NOT NULL,  
+  established_date  Date NOT NULL DEFAULT now(), 
+  primary_contact JSONB NOT NULL,
+  legal_authority_contact JSONB NOT NULL,  
+  institution_address JSONB NOT NULL,  
+  institution_mailing_address JSONB,
```

Comment:
This is no longer part of the requirement. I was not considering adding it right now on the UI.

Reply:
Lets discuss

Reply:
Done